src/wallet/rpcwallet.cpp

```diff
@@ -2664,6 +2664,31 @@ UniValue fundrawtransaction(const JSONRPCRequest& request)
     return result;
 }
 
+// Calculate the size of the transaction assuming all signatures are max size
+// Use DummySignatureCreator, which inserts 72 byte signatures everywhere.
```

Comment:
Might want to add an additional comment pointing out that ALL inputs MUST be in mapWallet (eg IsAllFromMe, which we do check in bumpfee).